src/Controller/MoviesController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
```

Comment:
It is good practice if you always add `declare(strict_types=1)` at the begining of your files. :slightly_smiling_face: